Ctor 875 plugin storage emc datadomain snmp misc enhancements (#3749)
centreon/centreon-documentation: pp/integrations/plugin-packs/procedures/hardware-storage-emc-datadomain-snmp.md

id: hardware-storage-emc-datadomain-snmp
title: EMC Data Domain
---
import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

## Connector dependencies

The following monitoring connectors will be installed when you install the **EMC Data Domain** connector through the
**Configuration > Monitoring Connector Manager** menu:
* [Base Pack](./base-generic.md)

## Pack assets

### Templates

The Monitoring Connector **EMC Data Domain** brings a host template:

* **HW-Storage-EMC-DataDomain-SNMP-custom**

The connector brings the following service templates (sorted by the host template they are attached to):

<Tabs groupId="sync">
<TabItem value="HW-Storage-EMC-DataDomain-SNMP-custom" label="HW-Storage-EMC-DataDomain-SNMP-custom">

| Service Alias   | Service Template                                      | Service Description                          | Discovery  |
|:----------------|:------------------------------------------------------|:---------------------------------------------|:----------:|
| Alerts          | HW-Storage-EMC-DataDomain-Alerts-SNMP-custom          | Check current alerts                         |            |

> | Cleaning        | HW-Storage-EMC-DataDomain-Cleaning-SNMP-custom        | Check last time filesystems had been cleaned |            |

Reviewer: ```suggestion
| Cleaning        | HW-Storage-EMC-DataDomain-Cleaning-SNMP-custom        | Check last time filesystems have been cleaned |            |
```

### Using a service template provided by the connector

1. If you have used a host template and checked **Create Services linked to the Template too**, the services linked to the template have been created automatically, using the corresponding service templates. Otherwise, [create manually the services you want](/docs/monitoring/basic-objects/services) and apply a service template to them.
2. Fill in the macros you want (e.g. to change the thresholds for the alerts). Some macros are mandatory (see the table below).

<Tabs groupId="sync">
<TabItem value="Alerts" label="Alerts">

| Macro                 | Description                                                                                           | Default value                                         | Mandatory   |
|:----------------------|:------------------------------------------------------------------------------------------------------|:------------------------------------------------------|:-----------:|

> | TRULYALERT            | Expression to define a truly alert (default: '%{severity} =~ /emergency\|alert\|warning\|critical/i') | %{severity} =~ /emergency\|alert\|warning\|critical/i |             |

Reviewer: "truly"

### Using a service template provided by the connector

1. If you have used a host template and checked **Create Services linked to the Template too**, the services linked to the template have been created automatically, using the corresponding service templates. Otherwise, [create manually the services you want](/docs/monitoring/basic-objects/services) and apply a service template to them.
2. Fill in the macros you want (e.g. to change the thresholds for the alerts). Some macros are mandatory (see the table below).

<Tabs groupId="sync">
<TabItem value="Alerts" label="Alerts">

| Macro                 | Description                                                                                           | Default value                                         | Mandatory   |
|:----------------------|:------------------------------------------------------------------------------------------------------|:------------------------------------------------------|:-----------:|

> | TRULYALERT            | Expression to define a truly alert (default: '%{severity} =~ /emergency\|alert\|warning\|critical/i') | %{severity} =~ /emergency\|alert\|warning\|critical/i |             |

Reviewer: ```suggestion
| TRULYALERT            | Expression to define an actual alert (default: '%{severity} =~ /emergency\|alert\|warning\|critical/i') | %{severity} =~ /emergency\|alert\|warning\|critical/i |             |
```
